src/scenes/Projects/Members/Update/UpdateProjectMember.tsx

```diff
@@ -66,10 +72,11 @@ export const UpdateProjectMember = ({
 
   const availableRoles = data?.user.roles.value ?? [];
   return (
-    <DialogForm<UpdateProjectMemberInput>
+    <DialogForm<FormShape>
       title="Update Team Member Role"
       closeLabel="Close"
       submitLabel="Save"
+      sendIfClean
```

Comment:
```suggestion
      sendIfClean="delete"
```
If there aren't role changes we should make an update call.